resources/definitions/arjunpro_mirrored.def.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version": 2,
+    "name": "Arjunpro 300 Mirror",
```

Comment:
You've spelled it `Arjunpro` here, but `Arjun Pro` earlier. I'd think the `Arjun Pro` name is more what most people would think the real name of the printer is, right?

Reply:
actually yes, but on printer make it was printed as arjun 300 pro instead arjun pro 300.
our team head forced us to change and to use like arjunpro 300 as definition /title .
most of our customers are from industries and colleges/institutes.

below are images of our arjun and arjun pro
![WhatsApp Image 2021-09-22 at 8 20 28 PM](https://user-images.githubusercontent.com/1712395/134366952-d28fda8b-954c-4dfe-a855-e32c02981c17.jpeg)
![20191106181942_2J2A0897 JPG](https://user-images.githubusercontent.com/1712395/134366964-3cab6b14-487f-40b4-9d75-493d2c84116d.jpg)
![WhatsApp Image 2021-09-22 at 8 12 58 PM](https://user-images.githubusercontent.com/1712395/134366989-b5cf448b-23e0-4924-9140-bb3365e8d190.jpeg)